src/components/OpacityView.js

```diff
@@ -49,11 +47,7 @@ function OpacityView(props) {
         }
     }, [props.shouldDim, props.dimmingValue, opacity]);
 
-    return (
-        <Animated.View style={[opacityStyle]}>
-            <View style={StyleUtils.parseStyleAsArray(props.style)}>{props.children}</View>
-        </Animated.View>
-    );
+    return <Animated.View style={[props.style, opacityStyle]}>{props.children}</Animated.View>;
```

Comment:
This is against the styling guide. We should still `parseStyleAsArray` and deconstruct the styles.
```js
<Animated.View style={[opacityStyle, ...StyleUtils.parseStyleAsArray(props.style)]}>{props.children}</Animated.View>;
```